build.sbt

```diff
@@ -16,9 +16,19 @@ def isCI = System.getenv("CI") != null
 def scala212 = "2.12.20"
 def scala213 = "2.13.16"
 
+def libJavaVer = sys.env.get("SCALAFMT_LIB_JAVA_RELEASE")
+def libJavaVerScalacOptions = libJavaVer
+  .fold(Seq.empty[String])(v => Seq("-release", v))
+def libJavaVerJavacOptions = libJavaVer
+  .fold(Seq.empty[String])(v => Seq("--release", v))
+
 inThisBuild {
   List(
     version ~= { dynVer =>
+      libJavaVer match {
+        case Some(v) => println(s"Forcing java release $v for library projects")
+        case None => println(s"Using current java release for library projects")
```

Comment:
looks like it's not working. in java11 tests, this is the message that is displayed (instead of "Forcing java release 8..." as I expected). am i setting `env` incorrectly in `.yml`?

Reply:
ok, `env` is not propagated for some reason, so hardcoded it for those three projects which are exposed as libraries. didn't touch the rest.